cogs/reward.py

```diff
@@ -11,66 +11,123 @@ def __init__ (self, bot):
     @commands.command()
 
     async def reward(self,ctx, timeString=None, tier=None):
-        rewardCommand = f"\nPlease follow this format:\n```yaml\n{commandPrefix}reward \"#h#m\" \"tier\"```\n"
+        rewardCommand = f"\nPlease follow this format:\n```yaml\n{commandPrefix}reward \"#h#m\" \"tier or character name\"```\n"
 
         def convert_to_seconds(s):
             return int(s[:-1]) * seconds_per_unit[s[-1]]
 
         channel = ctx.channel
+        author = ctx.author
+        charEmbed = discord.Embed()
+        charEmbedmsg = None
+        char = tier #this is mainly for my sanity
+        
         if timeString is None:
             await channel.send(content="Woops, you're forgetting the time duration for the command." + rewardCommand)
             return
 
         if tier is None:
-            await channel.send(content="Woops, you're forgetting the tier for the command. Please try again with 1, 2, 3, or 4 or Junior, Journey, Elite, or True as the tier, and" + rewardCommand)
+            await channel.send(content="Woops, you're forgetting the tier or character name for the command. Please try again with 1, 2, 3, or 4 or Junior, Journey, Elite, or True as the tier, or use a character name and" + rewardCommand)
             return
 
         seconds_per_unit = { "m": 60, "h": 3600 }
         lowerTimeString = timeString.lower()
+        
+        characterPresent = None
```

Comment:
I don't see the characterPresent = False case being used anywhere, instead of differentiating it with None, just use False